packages/autorest.go/test/acr/azacr/zz_containerregistry_client.go

```diff
@@ -430,8 +430,10 @@ func (client *ContainerRegistryClient) NewGetManifestsPager(name string, options
 		Fetcher: func(ctx context.Context, page *ContainerRegistryClientGetManifestsResponse) (ContainerRegistryClientGetManifestsResponse, error) {
 			ctx = context.WithValue(ctx, runtime.CtxAPINameKey{}, "ContainerRegistryClient.NewGetManifestsPager")
 			nextLink := ""
-			if page != nil {
+			if page != nil && page.Link != nil {
```

Comment:
n.b. if everything is a pointer, it's best to check that we're not about to dereference a `nil`. Not really related to this PR, but good to have anyway.